AirCasting/CreateSessionViews/ConfirmCreatingSessionView.swift

```diff
@@ -16,66 +16,63 @@ struct ConfirmCreatingSessionView: View {
             isPresentingAlert = error != nil
         }
     }
-
```

Comment:
The only modifications in this file are formatting ones, we should avoid that (due to conflicts)